apps/ledger-live-desktop/src/newArch/Collectibles/components/DetailDrawer/components/DetailField.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { memo } from "react";
+import { Skeleton } from "LLD/Collectibles/components/Skeleton";
+import { DetailFieldProps } from "LLD/Collectibles/types/DetailDrawer";
+import { CopyableField } from ".";
+import Text from "~/renderer/components/Text";
+import styled from "styled-components";
+import { SplitAddress } from "~/renderer/components/OperationsList/AddressCell";
+import { useTranslation } from "react-i18next";
+
+const Separator = styled.div`
+  width: 100%;
+  height: 1px;
+  background-color: ${({ theme }) => theme.colors.palette.text.shade10};
+  margin: 24px 0px;
+`;
+
+const Container = styled.div<{ isHorizontal?: boolean }>`
+  width: 100%;
+  display: flex;
+  flex-direction: ${({ isHorizontal }) => (isHorizontal ? "row" : "column")};
+  justify-content: space-between;
+`;
+
+const ValueBox = styled.div<{ isHorizontal?: boolean }>`
+  width: 100%;
+  align-self: end;
+  display: ${({ isHorizontal }) => (isHorizontal ? "flex" : "block")};
+  justify-content: ${({ isHorizontal }) => (isHorizontal ? "flex-end" : "flex-start")};
+`;
+
+const Pre = styled.span`
+  white-space: pre-line;
+  display: block;
+  unicode-bidi: embed;
+  line-break: anywhere;
+  line-height: 15px;
+`;
+
+const HashContainer = styled.div`
+  word-break: break-all;
+  user-select: text;
+  width: 100%;
+  min-width: 100px;
+`;
+
+const DetailFieldComponent: React.FC<DetailFieldProps> = ({
+  label,
+  value,
+  hasSeparatorTop,
+  hasSeparatorBottom,
+  isHorizontal,
+  isCopyable,
+  isLoading,
+  isHash,
+}: DetailFieldProps) => {
+  const { t } = useTranslation();
+
+  if (!value) return null;
```

Comment:
why return null here and below check if `{ value ? (Component) : null}`?

Reply:
To avoid to render the section title. I thought it would be cleaner to make a small return null here rather than encompass the whole component